### ISSUE TITLE
Whatsapp instance gets stuck on infinite loading screen

### ISSUE THREAD

**Issue by faradaytrs** [orkestral/venom#1959]
## Description

After successful login into whatsapp account, after 2-3 minutes account gets stuck on infinite loading screen:
![下载](https://user-images.githubusercontent.com/3426470/199194682-3f33d3ca-b2c3-4779-9a4f-3e931923ff4f.jpeg)

[Description of the bug, When Issue Happens]

## Environment

- **Venom version(s):*master branch* [e.g. 2.0.27, 2.2.1]
- **Browser:*Chrome 108* [e.g. Chrome 87, Chromium 85]
- **OS:*Linux ubuntu 20.04 lts* [e.g. OSX 10.13.4, Windows 10]
- **Node version:*18 [active lts]* [e.g. Node 8, Node 14]

## Steps to Reproduce

1. Login
2. Wait few minutes
3. Account becomes unresponsibe and on screenshots you can see that it's stuck in infinite loading.

## Log Output

```
globus_chat-api.1.kcdcicd3ly77@ls3    | - [instance: 195837]: Checking is logged...
globus_chat-api.1.kcdcicd3ly77@ls3    | - [instance: 213509]: Checking is logged...
globus_chat-api.1.kcdcicd3ly77@ls3    | - [instance: julia]: Checking is logged...
```

## Your Code
i use following settings for client:

```
{
            logQR: false,
            autoClose: 0,
            addBrowserArgs: ["--no-sandbox", "--disable-dev-shm-usage"],
            disableSpins: true,
            disableWelcome: true,
            useChrome: true,
            headless: true
}
```

## Additional context / Screenshot

Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by faradaytrs** [comment]
Actually this issue is really annoying, i would like to know if anybody else experienced it

**Comment by luthfifadle** [comment]
try delete 'tokens' folder and scan the qr again

**Comment by faradaytrs** [comment]
> try delete 'tokens' folder and scan the qr again

it works but it gets broken again quickly

**Comment by faradaytrs** [comment]
![IMAGE 2022-11-03 12:18:23](https://user-images.githubusercontent.com/3426470/199646763-af80c30c-5120-4e17-854d-e2a86d056f8a.jpg)
i found out that it's related to calling this getAllChats function...

**Comment by orkestral** [comment]
solved #[54d5a7b](https://github.com/orkestral/venom/commit/54d5a7b2cf4aa9523dfe59efde8abed6418d9707)
new version 3.4.1 for update
```javascript
npm i venom-bot@latest
```
